src/accelerate/tracking.py

```diff
@@ -681,12 +685,187 @@ def finish(self):
         mlflow.end_run()
 
 
+class ClearMLTracker(GeneralTracker):
+    """
+    A `Tracker` class that supports `clearml`. Should be initialized at the start of your script.
+
+    Environment:
+    - **CLEARML_PROJECT** (`str`, *optional*) - The default ClearML project name. Can be overwritten by setting
+      `project_name` in `task_init_kwargs`.
+    - **CLEARML_TASK** (`str`, *optional*) - The default ClearML task name. Can be overwritten by setting `task_name`
+      in `task_init_kwargs`.
+
+    Args:
+        run_name (`str`, *optional*):
+            Name of the experiment. If ClearML's `Task.init`'s `task_name` and `project_name` are not specified in
+            kwargs, they will default to this value
```

Comment:
```suggestion
        run_name (`str`, *optional*):
            Name of the experiment. By default will try and pull from either `CLEARML_PROJECT` or `CLEARML_TASK` from the environment if set.
```

Reply:
In general let's try to prioritize the explicit config first, then fall back on the environmental variables that a user would directly know.

Reply:
done